src/MentionTracker.ts

```diff
@@ -0,0 +1,26 @@
+import Discord = require("discord.js");
+
+export default class MentionTracker {
+    private bot: Discord.Client;
+
+
+    constructor(bot: Discord.Client) {
+        console.log("Requested Mention Tracking extension..");
+        this.bot = bot;
+
+        this.bot.on("message", this.onMention.bind(this));
+    }
+
+    onMention(message: Discord.Message) {
```

Comment:
How does this handle a Rioter mentioning another Rioter? I haven't tested it, but it looks like it would do something like this:
Rioter1: Hi @Rioter2
Botty McBotface: @Rioter1, please do not ping the rioters!
Botty McBotface: @Botty McBotface, please do not ping the rioters!

Additionally, I'm not sure how I feel about this feature as a whole. It's okay to ping Rioters if they've asked you to do so, or you're replying to them.